portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,12 +33,16 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import com.google.appengine.api.datastore.FetchOptions;
+import java.util.logging;
 
 // Servlet that returns some example content. 
 @WebServlet("/comments")
 public class DataServlet extends HttpServlet {
   
   private final int DEFAULT_QUANTITY = 10;
+  private final String DEFAULT_LANGUAGE = "en";
+  private final String DEFUALT_COMMENT = "--";
+  Logger logger = Logger.getLogger(DataServlet.class.getName());
```

Comment:
You should be able to make this private.

Reply:
Done!